huaweicloud/services/cnad/data_source_huaweicloud_cnad_advanced_available_objects.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
 )
 
+// @API CNAD GET /v1/cnad/packages/{package_id}/unbound-protected-ips
```

Comment:
the product should be **AAD** for all cnad resources and data sources.

Reply:
done